core/consensus/common/Cargo.toml

```diff
@@ -20,3 +20,7 @@ parity-codec = { version = "3.3", features = ["derive"] }
 
 [dev-dependencies]
 test_client = { package = "substrate-test-client", path = "../../test-client" }
+
+[features]
+default = []
+test-helpers = []
```

Comment:
That is a nice idea, but given [this](https://github.com/rust-lang/cargo/issues/1796), it has no effect :( `test-helpers` will be enabled always, especially in a workspace like Substrate.

Reply:
Oh, thanks - haven't noticed that. I'm not sure - whether it is better to leave it under non-working-feature-gate or remove the feature at all?

Reply:
Do you think that this decreases the performance of the production code? I think we should keep the feature and hope for someday where they finally fix it in cargo..

Reply:
It shouldn't affect production code at all (except for additional match pattern checking cost). OK - I'll leave it here